### PR TITLE
Curb flip spam

### PR DESCRIPTION
**Pull request by coldud13**
## About The Pull Request
The *flip emote now has a 20 percent chance to fail and land on your face (which hurts).

## Why It's Good For The Game
Seeing people flip at 5000RPM looks bad.

## Changelog
:cl:
balance: Flip spam begone
/:cl:

### CODE REVIEW

**Comment by SquishyOranges** [comment]
This seems unnecessary.

File: modular_skyrat/code/modules/mob/emote.dm [hunk 1]
```diff
@@ -58,3 +58,13 @@
...
 		message = "makes a very loud noise."
 	. = ..()
+
+/datum/emote/flip/run_emote(mob/living/user, params) //no fun allowed :)
+	if(prob(20))
+		user.visible_message("<span class='warning'>[user] tries to flip, but lands flat on their face!</span>", "<span class='danger'>You try to flip, but land flat on your face!</span>")
+		user.Knockdown(20)
+		user.setDir(NORTH)
+		user.apply_damage(rand(1, 8), BRUTE, BODY_ZONE_HEAD)
+		user.adjustOrganLoss(ORGAN_SLOT_BRAIN, rand(1, 5), 20)
```
Comment (Afya212) [inline modular_skyrat/code/modules/mob/emote.dm:68]: Brain damage here is definitely excessive.

Comment (Afya212) [inline modular_skyrat/code/modules/mob/emote.dm:68]:
```suggestion
```
Having brain damage here is definitely excessive.

Comment (jjpark-kb) [reply modular_skyrat/code/modules/mob/emote.dm:68]: brain damage actually gets healed over time, and its capped at 20, which doesnt leave any permanent brain damage.

**Comment by IronEleven** [comment]
This is a huge DEsword nerf.

**Comment by Investigator77** [comment]
Literally made the desword useless lol

**Pull request merged**